src/utils/dataJSON/helper/helper.ts

```
@@ -105,9 +105,16 @@ export function getPreviewDetails(applicationData: any) {
   }
 
   for (const key in applicationData) {
+    const hiddenKey = [
+      "samagraId",
+      "currentSchoolName",
+      "currentSchoolAddress",
+      "currentSchoolAddressDistrict",
+    ];
     if (applicationData.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)